test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -150,57 +151,24 @@ public static WebArchive deploy() {
     public void lraCoordinatorShortTimeoutLRA(@ArquillianResource @OperateOnDeployment(LRA_PARTICIPANT_DEPLOYMENT_QUALIFIER) URL baseURL)
             throws URISyntaxException {
 
-        String lraId;
-        URI lraListenerURI = UriBuilder.fromUri(baseURL.toURI()).path(LRAListener.LRA_LISTENER_PATH).build();
-
-        // Starts an LRA with a short time limit by invoking a resource annotated with @LRA. The time limit is
-        // defined as LRAListener.LRA_SHORT_TIMELIMIT
-        Response response = client
-                .target(lraListenerURI)
-                .path(LRAListener.LRA_LISTENER_ACTION)
-                .request()
-                .put(null);
-
-        Assert.assertEquals("LRA participant action", 200, response.getStatus());
+        Long SHORTER_TIMEOUT = 1000L;
+        URI shortLRA = lraClient.startLRA(null, "Short Timeout Recovery Test", SHORTER_TIMEOUT, ChronoUnit.MILLIS);
 
+        // The LRA transaction could have been started via lraClient but it is useful to
```

Comment:
This statement is a little confusing because on the new line 155 lraClient is being used

Reply:
Actually, I see where it was moved from in the old diff and my thought is that the intention of the test before was to use the annotated client directly. I am not -1 to change to using lraClient (as long as we have tests for the filters elsewhere) but it seems like this comment is no longer appropriate?

Reply:
OK, thank you!